lang/en.ts

```diff
@@ -8,7 +7,7 @@ import {
 } from '../lib/ui';
 import { getProjectSettingsUrl } from '../lib/projects/urls';
 
-type LangFunction = (...args: (string | number)[]) => string;
+type LangFunction = (...args: never[]) => string;
```

Comment:
I updated this to never because it's impossible and know all of the permutations of possible arguments, and as far as the satisfies is concerned we don't really care the shape of the arguments, just just care that it is a function that returns a string